### PR TITLE
Tooltips: Strong Winds boost Weather Ball's power

### PR DESCRIPTION
**Pull request by MacChaeger**

### CODE REVIEW

**Comment by urkerab** [comment]
At this point that covers every type of weather, so wouldn't just testing `this.battle.weather` directly suffice?

**Comment by Zarel** [comment]
urkerab makes a strong point.

**Comment by MacChaeger** [comment]
One wonders why that wasn't done in the first place...

**Comment by MacChaeger** [comment]
Well, probably because whoever did it didn't realize that strong winds boosts weather ball; otherwise they would have included it, probably.

File: js/client-battle-tooltips.js [hunk 1]
```diff
@@ -965,7 +965,7 @@ var BattleTooltips = (function () {
 			if (move.id in table.overrideBP) basePower = table.overrideBP[move.id];
 		}
 		var basePowerComment = '';
-		var thereIsWeather = (this.battle.weather in {'sunnyday': 1, 'desolateland': 1, 'raindance': 1, 'primordialsea': 1, 'sandstorm': 1, 'hail':1});
+		var thereIsWeather = this.battle.weather;
```
Comment (Zarel) [inline js/client-battle-tooltips.js]: `!!`

Comment (Zarel) [inline js/client-battle-tooltips.js]: (Cast to boolean)

Comment (MacChaeger) [inline js/client-battle-tooltips.js]: Oh! I thought you were just expressing alarm of some kind.

Comment (Zarel) [inline js/client-battle-tooltips.js]: lol, yeah, in hindsight I realized that was unclear.

**Pull request merged**